src/Pages/Detail/ColorList.scss

```
@@ -9,6 +9,12 @@
       animation: onHoverAtag 0.25s ease-out;
       animation-fill-mode: both;
     }
+    .checker{
+      width: 32px;
+    }
+    .hidden{
+      display: none;
+    }
```

```suggestion
    .checker {
      width: 32px;
    }
    
    .hidden {
      display: none;
    }
```
기본적인 컨벤션 꼭 지켜주세요!
- 중괄호 앞에 한칸 띄우기
- 새로운 셀렉터 앞에 한줄 띄우기